lib/config/router.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+// router.config options
+type RouterConfig struct {
+  NetDbDir string
```

Comment:
Is the goal to be compatible with I2P's directory structure (so either application could be ran from the same configuration directory)?  That would be great, but if not perhaps we could explore alternative storage (sqlite?)